### DISCUSSION TITLE
Restoring/starting emacs process buffers?

### DISCUSSION THREAD

**Discussion by WorldsEndless** [alphapapa/burly.el#66] [Ideas]
I am an EXWM user with 3 physical monitors and make heavy use of splits and external (usually FireFox) buffers, as well as emacs sub-processes in which I run elfeed and another for telega (both of which have caused troubles when executed through my main process). I realize that the nature of this problem, starting processes that are all external to the given emacs instance, probably takes this out of the range of Burley. But this seems as good a place as any to ask if a known solution exists. Is there a desktop/bookmarking system that can start external processes?

**Comment by alphapapa** [comment]
I don't know of such a system.  Maybe you could come up with something.  I probably couldn't offer much support for it, myself, due to the complexities of using EXWM.  But if you need something like an extra hook function or frame attribute, that might be doable in Burly.

**Comment by WorldsEndless** [comment]
I think we can abstract it away from the complications of exwm. Is there anything known about restoring external processes of the sort started with the built-in `async-shell-command` (M-&)?
